pom.xml

```diff
@@ -66,6 +66,16 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 	</properties>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>ch.qos.logback</groupId>
+				<artifactId>logback-classic</artifactId>
+				<version>1.2.3</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>

```

Comment:
I am ok with adding a specific logger implementation for test cases and for examples. But not here in parent pom

Reply:
Please check it once again: it is only dependencyManagement, not a dependecy included